colors/gruvbox.vim

```diff
@@ -815,6 +815,11 @@ hi! link LspDiagnosticsUnderlineHint GruvboxAquaUnderline
 
 " }}}
 
+" Treesitter: {{{
+hi! link TSKeywordOperator GruvboxRed
```

Comment:
I picked red so that the highlighting for keyword operators (`and`, `or`, `not`, etc.) matches the highlighting for keywords (`if`, `for`, etc.).